src/python/devcontainer-feature.json

```diff
@@ -65,15 +65,7 @@
         "ms-python.vscode-pylance"
       ],
       "settings": {
-        "python.defaultInterpreterPath": "/usr/local/python/current/bin/python",
-        "python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
-        "python.formatting.blackPath": "/usr/local/py-utils/bin/black",
```

Comment:
Looks like we'd need similar changes to the images repo https://github.com/search?q=repo%3Adevcontainers%2Fimages+python.formatting+python.linting&type=code

Reply:
yes, good point! I'll send a PR there too 😊